as-controller-board-status/functions/models.go

```diff
@@ -70,6 +69,7 @@ type CheckBoardStatus struct {
 	Configuration                             *config.ControllerBoardStatusConfig
 	SubscriptionClient                        interfaces.SubscriptionClient
 	NotificationClient                        interfaces.NotificationClient
+	CommandClient                             interfaces.CommandClient
```

Comment:
This is never set. Should be done in main similar to NotificationClient
https://github.com/intel-retail/automated-vending/blob/Edgex-3.0/as-controller-board-status/main.go#L83-L89

Reply:
hey this is a good point: actually originally i thought the commandClient could be initialized on the fly once it is put into the configuration.yaml file; but looks like only core-data/metadata done that automatically

Reply:
> hey this is a good point: actually originally i thought the commandClient could be initialized on the fly once it is put into the configuration.yaml file; but looks like only core-data/metadata done that automatically

Great catch, yes the command client config also needs to be added to the configuration.yaml and an override added to the compose file.

Reply:
@presatish,  Looks like the configuration.yaml is missing notification client config. Please add. @seanohair22 can make sure the proper override are in the compose file.

Reply:
I had added it before but looks like I deleted that branch and forgot to add in this one :-)